presets/4.3/rates/mouseFreestyle.txt

```diff
@@ -0,0 +1,39 @@
+#$ TITLE: mouseFPV Freestyle Rates
+#$ FIRMWARE_VERSION: 4.3
+#$ CATEGORY: RATES
+#$ STATUS: COMMUNITY
+#$ KEYWORDS: freestyle
+#$ AUTHOR: mouseFPV
+#$ DESCRIPTION: These are the freestyle rates used by Youtube Freestyler and tuner "mouseFPV"
+#$ DESCRIPTION: https://www.youtube.com/c/mouseFPV
```

Comment:
for the proper link it should be something like:
`[mouseFPV YouTube channel](https://www.youtube.com/c/mouseFPV)`

Reply:
`#$ PARSER: MARKED` will enable markdown rendering, so we can format text just like in the Github comments right here. 
You can even use html and insert your logo on top of the text with a hyperlink to your youtube channel, etc...
Look at the [HDZero VTX preset](https://github.com/betaflight/firmware-presets/pull/246) as an example.